README.md

```diff
@@ -75,10 +75,12 @@ The script will
 - `yarn install` within each subdirectory
 - Updates your local `/etc/hosts` to support the local DNS name redirect
 'local.rosalution.cgds' to localhost.
+- creates a virtual environment for Python called "rosalution_env" within the backend directory and activates it.
+- installs Python dependencies within the virtual environment and then deactivates the virtual environment.
+    - use `source backend/rosalution_env/bin/activate` to activate the virtual environment.
```

Comment:
You can move this line to be in the Python README.  

We can also add an echo output in the script notifying the user that they can activate the virtual environment via `source backend/rosalution_env/bin/activate`.

Reply:
I believe I have now made the requested changes.